reference/cli/wpe/README.md

```diff
@@ -3,14 +3,16 @@
 This section contains reference information on using WP Engine's command line
 client. Each command has a reference page along with samples.
 
+**NOTE:** With version 0.1.4 released March 18, 2021, Atlas CLI no longer requires the `alpha` syntax (such as `wpe alpha apps`of previous versions. If you are still using a version of Atlas CLI, run `npm i @wpengine/headless-cli -g` to use commands without `alpha`.

```

Comment:
Isn't it missing "previous" or "older" in "If you are still using a version of Atlas CLI" though?